nalgebra-sparse/example/spmm.rs

```diff
@@ -0,0 +1,25 @@
+extern crate nalgebra_sparse;
+use nalgebra_sparse::CsrMatrix;
+use std::time::{Duration, Instant};
+
+#[cfg(feature = "io")]
+use nalgebra_sparse::io::load_coo_from_matrix_market_file;
+fn main() {
+    #[cfg(feature = "io")]
+    {
+        let sparse_input_matrix =
+            load_coo_from_matrix_market_file::<f64, _>("./data/crankseg_1.mtx").unwrap();
```

Comment:
This assumes you create a directory named `data` in the nalgebra-sparse directory, and then put the `crankseg_1` matrix in that directory. This matrix is a part of the open source SuiteSparse matrix dataset that is commonly used for performance benchmarking.